appserver/packager/external/jakarta-ee9-shim/pom.xml

```diff
@@ -119,8 +125,13 @@
                                     jakarta.enterprise.cdi-api,
                                     jakarta.interceptor-api,
                                     jakarta.xml.bind-api,
+                                    org.eclipse.persistence.core,
+                                    org.eclipse.persistence.moxy,
+                                    jakarta.interceptor-api,
```

Comment:
duplicate bundle name for interceptor, I think eclipselink and Moxy are not necessary after ugprade to 4.0 that we completed in the meantime.